common/config/azure-pipelines/integration-validation.yaml

```diff
@@ -31,9 +31,9 @@ jobs:
         Linux_node_14_x:
           imageName: ubuntu-latest
           nodeVersion: 14.x
-        # linux_node_16_x:
-        #   imageName: ubuntu-latest
-        #   nodeVersion: 16.x
+        linux_node_16_x:
```

Comment:
Previously the latest node version was commented out. I've blindly followed the pattern, uncommenting 16.x and omitting 18.x.

What was the intention here?

Reply:
@skirby1996 any idea?

Reply:
It's limited due to running the latest version in the PR validation, `integration-pr-validation`. We didn't want to duplicate that within the scheduled run so it's over there instead.  This is the correct change IMO.